### PR TITLE
Change all instances of logos to include alt text 

### PR DESCRIPTION
**Pull request by Zineb-Ada**
This update the instances of logos, for all `academic users/institutions files`,  to include a record `logo` with `path` and `alt`.
This PR responds to issue #96.

### CODE REVIEW

**Comment by tmattio** [comment]
Thanks @Zineb-Ada!

Following your PR, I read W3C recommendations on alt text for images here: https://www.w3.org/WAI/tutorials/images/. It seems that for images that are purely decorative, they recommend to put an empty alt text (`alt=""`). I think for the industrial users and academic users page, the logo would qualify as purely decorative (indeed, the link with to the institution with its name is found just below the logo, so a screen reader would read, for instance, "Universidade da Beira Interior Universidade da Beira Interior logo", which is probably not a great experience)

I'm wondering if we should review the images on the site first, and list the ones that would need to have an alt text, and the ones where an empty alt text is justified.

PS: apologies for only reading the recommendations after you've done the work. FWIW, the implementation in the PR looks very good!

**Comment by Zineb-Ada** [comment]
Thank you @tmattio for your well argued comments. OK it's noted.

**Comment by cuihtlauac** [comment]
I had a discussion with @shindere about what to put inside `alt`. He's told me `alt` MUST be used only when the purpose of an image is to convey text; in that case, `alt` MUST contain the text displayed by the image. Otherwise, it's decorative, and `alt` MAY be empty.

I think we should double-check with him on logos containing text.

**Comment by cuihtlauac** [comment]
@Zineb-Ada: here is a PR on https://github.com/Zineb-Ada/ocaml.org/pull/1 (how nice to be first!) w.r.t. this branch. It reduces the size of the diff. Appart from that and the remarks on the alt contents, this goes in the right direction. You can proceed with the other kinds of data which need to receive the same treatment.

**Comment by Zineb-Ada** [comment]
@cuihtlauac yes good idea. I will check with @shindere and fix this issue ASAP.

**Comment by shindere** [comment]
Cuihtlauac Alvarado (2023/01/05 23:45 -0800):
> I had a discussion with @shindere about what to put inside alt. He's
> told me alt should be used only when the purpose of an image is to
> convey text,

More precisely, when the image conveys *information*.

> in that case, alt should contain the text displayed by the image.
> Otherwise, it's decorative and alt should be empty.

In fact the rule of thumb is that all the images must have an
alternative text. If the image is decorative (does not convey any
information) then that alternative text must be empty so that the
assistive technologies can properly ignore this image. If the image
carries information then the alternative text must be non-empty.

Deciding whether an image is either decorative or carries information is
an editorial choice.

For those interested, the explanations are here:

https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/

See the first section about images.

If you wish I can ask real accessibility experts about how to deal with
logos. Just let me know.

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 1]
```diff
@@ -39,34 +39,34 @@ Layout.render
       </div>
     </div>
     <div :class="continent != 'All' ? 'hidden' : 'grid-logos'">
-      <% users |> List.iter (fun (item : Ood.Academic_institution.t) -> let logo = match item.logo with | Some x -> "/media/" ^ x | None -> "" in %>
+      <% users |> List.iter (fun (item : Ood.Academic_institution.t) -> match item.logo with | None -> () | Some { path; alt } -> %>
       <a href="<%s item.url %>" target="_blank" class="p-12 hover:text-primary-600 flex items-center flex-col">
-        <img src="<%s logo %>" class="m-auto" alt="<%s item.name %> logo" />
+        <img src="<%s "/media/" ^ path %>" class="m-auto" alt="<%s alt %>" />
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]: " class="m-auto" alt="<%s alt %>" />

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 2]
```diff
@@ -39,34 +39,34 @@ Layout.render
...
       </a>
       <% ); %>
     </div>
 
     <div :class="continent != 'North America' ? 'hidden' : 'grid-logos'">
-      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "North America") |> List.iter (fun (item : Ood.Academic_institution.t) -> let logo = match item.logo with | Some x -> "/media/" ^ x | None -> "" in %>
+      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "North America") |> List.iter (fun (item : Ood.Academic_institution.t) -> match item.logo with | None -> () | Some { path; alt } -> %>
       <a href="<%s item.url %>" target="_blank" class="p-12 hover:text-primary-600 flex items-center flex-col">
-        <img src="<%s logo %>" class="m-auto" alt="<%s item.name %> logo" />
+        <img src="<%s "/media/" ^ path %>" class="m-auto" alt="<%s alt %>" />
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]: " class="m-auto" alt="<%s alt %>" />

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 3]
```diff
@@ -39,34 +39,34 @@ Layout.render
...
         <div class="font-bold mt-8"><%s item.name %></div>
       </a>
       <% ); %>
     </div>
     <div :class="continent != 'Europe' ? 'hidden' : 'grid-logos'">
-      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "Europe") |> List.iter (fun (item : Ood.Academic_institution.t) -> let logo = match item.logo with | Some x -> "/media/" ^ x | None -> "" in %>
+      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "Europe") |> List.iter (fun (item : Ood.Academic_institution.t) -> match item.logo with | None -> () | Some { path; alt } -> %>
       <a href="<%s item.url %>" target="_blank" class="p-12 hover:text-primary-600 flex items-center flex-col">
-        <img src="<%s logo %>" class="m-auto" alt="<%s item.name %> logo" />
+        <img src="<%s "/media/" ^ path %>" class="m-auto" alt="<%s alt %>" />
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]: " class="m-auto" alt="<%s alt %>" />

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 4]
```diff
@@ -39,34 +39,34 @@ Layout.render
...
         <div class="font-bold mt-8"><%s item.name %></div>
       </a>
       <% ); %>
     </div>
     <div :class="continent != 'Asia' ? 'hidden' : 'grid-logos'">
-      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "Asia") |> List.iter (fun (item : Ood.Academic_institution.t) -> let logo = match item.logo with | Some x -> "/media/" ^ x | None -> "" in %>
+      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "Asia") |> List.iter (fun (item : Ood.Academic_institution.t) -> match item.logo with | None -> () | Some { path; alt } -> %>
       <a href="<%s item.url %>" target="_blank" class="p-12 hover:text-primary-600 flex items-center flex-col">
-        <img src="<%s logo %>" class="m-auto" alt="<%s item.name %> logo" />
+        <img src="<%s "/media/" ^ path %>" class="m-auto" alt="<%s alt %>" />
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]: " class="m-auto" alt="<%s alt %>" />

File: tool/ood-gen/lib/academic_institution.ml [hunk 5]
```diff
@@ -85,7 +93,7 @@ let pp ppf v =
   ; body_html = %a
   }|}
     Pp.string v.name Pp.string (Utils.slugify v.name) Pp.string v.description
-    Pp.string v.url (Pp.option Pp.string) v.logo Pp.string v.continent
+    Pp.string v.url (Pp.option pp_logo) v.logo Pp.string v.continent
```
Comment (cuihtlauac) [inline tool/ood-gen/lib/academic_institution.ml]: Pp.string v.url (Pp.option ("media" ^ pp_logo) v.logo Pp.string v.continent

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

File: tool/ood-gen/lib/academic_institution.ml (hunk 5 above)
Comment (Zineb-Ada) [reply tool/ood-gen/lib/academic_institution.ml]: I don't think I can do this, however I can concatenate `/media/` and `v.path` directly in `pp_logo` function : 
`
let pp_logo ppf (v : logo) =
  Fmt.pf ppf {|
  { path = %a
  ; alt = %a
  }|} Pp.string ("/media/" ^ v.path) (Pp.option Pp.string)
    v.alt
`

**Comment by Zineb-Ada** [comment]
In this commit [64dd3a6](https://github.com/ocaml/ocaml.org/pull/760/commits/64dd3a68c8692b6cfe7a98dc1163092ba5f57c9e) i removed `alt` from the `academic_institutions*.md` files, changed them to string option in the `academic_institution.ml` and updated `academic_users.eml`.

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 6]
```diff
@@ -39,34 +39,38 @@ Layout.render
       </div>
     </div>
     <div :class="continent != 'All' ? 'hidden' : 'grid-logos'">
-      <% users |> List.iter (fun (item : Ood.Academic_institution.t) -> let logo = match item.logo with | Some x -> "/media/" ^ x | None -> "" in %>
+      <% users |> List.iter (fun (item : Ood.Academic_institution.t) -> match item.logo with | None -> () | Some { path; alt } ->
+      let alt = match alt with | Some x -> x | None -> "" in %>
       <a href="<%s item.url %>" target="_blank" class="p-12 hover:text-primary-600 flex items-center flex-col">
-        <img src="<%s logo %>" class="m-auto" alt="<%s item.name %> logo" />
+        <img src="<%s path %>" class="m-auto" alt="<%s alt %>" />
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
        <img src="<%s path %>" class="m-auto" alt="<%s Option.value ~default:"" alt %>" />
```

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 7]
```diff
@@ -39,34 +39,38 @@ Layout.render
       </div>
     </div>
     <div :class="continent != 'All' ? 'hidden' : 'grid-logos'">
-      <% users |> List.iter (fun (item : Ood.Academic_institution.t) -> let logo = match item.logo with | Some x -> "/media/" ^ x | None -> "" in %>
+      <% users |> List.iter (fun (item : Ood.Academic_institution.t) -> match item.logo with | None -> () | Some { path; alt } ->
+      let alt = match alt with | Some x -> x | None -> "" in %>
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
      <% users |> List.iter (fun (item : Ood.Academic_institution.t) -> match item.logo with | None -> () | Some { path; alt } -> %>
```

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 8]
```diff
@@ -39,34 +39,38 @@ Layout.render
...
+        <img src="<%s path %>" class="m-auto" alt="<%s alt %>" />
         <div class="font-bold mt-8"><%s item.name %></div>
       </a>
       <% ); %>
     </div>
 
     <div :class="continent != 'North America' ? 'hidden' : 'grid-logos'">
-      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "North America") |> List.iter (fun (item : Ood.Academic_institution.t) -> let logo = match item.logo with | Some x -> "/media/" ^ x | None -> "" in %>
+      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "North America") |> List.iter (fun (item : Ood.Academic_institution.t) -> match item.logo with | None -> () | Some { path; alt } ->
+      let alt = match alt with | Some x -> x | None -> "" in %>
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "North America")
               |> List.iter (fun (item : Ood.Academic_institution.t) -> match item.logo with | None -> () | Some { path; alt } -> %>
```

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 9]
```diff
@@ -39,34 +39,38 @@ Layout.render
...
       </a>
       <% ); %>
     </div>
     <div :class="continent != 'Europe' ? 'hidden' : 'grid-logos'">
-      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "Europe") |> List.iter (fun (item : Ood.Academic_institution.t) -> let logo = match item.logo with | Some x -> "/media/" ^ x | None -> "" in %>
+      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "Europe") |> List.iter (fun (item : Ood.Academic_institution.t) -> match item.logo with | None -> () | Some { path; alt } ->
+        let alt = match alt with | Some a -> a | None -> "" in %>
       <a href="<%s item.url %>" target="_blank" class="p-12 hover:text-primary-600 flex items-center flex-col">
-        <img src="<%s logo %>" class="m-auto" alt="<%s item.name %> logo" />
+        <img src="<%s path %>" class="m-auto" alt="<%s alt %>" />
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
        <img src="<%s path %>" class="m-auto" alt="<%s Option.value ~default:"" alt %>" />
```

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 10]
```diff
@@ -39,34 +39,38 @@ Layout.render
...
       </a>
       <% ); %>
     </div>
     <div :class="continent != 'Asia' ? 'hidden' : 'grid-logos'">
-      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "Asia") |> List.iter (fun (item : Ood.Academic_institution.t) -> let logo = match item.logo with | Some x -> "/media/" ^ x | None -> "" in %>
+      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "Asia") |> List.iter (fun (item : Ood.Academic_institution.t) -> match item.logo with | None -> () | Some { path; alt } ->
+      let alt = match alt with | Some x -> x | None -> "" in %>
       <a href="<%s item.url %>" target="_blank" class="p-12 hover:text-primary-600 flex items-center flex-col">
-        <img src="<%s logo %>" class="m-auto" alt="<%s item.name %> logo" />
+        <img src="<%s path %>" class="m-auto" alt="<%s alt %>" />
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
        <img src="<%s path %>" class="m-auto" alt="<%s Option.value ~default:"" alt %>" />
```

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 11]
```diff
@@ -39,34 +39,38 @@ Layout.render
...
       <% ); %>
     </div>
 
     <div :class="continent != 'North America' ? 'hidden' : 'grid-logos'">
-      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "North America") |> List.iter (fun (item : Ood.Academic_institution.t) -> let logo = match item.logo with | Some x -> "/media/" ^ x | None -> "" in %>
+      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "North America") |> List.iter (fun (item : Ood.Academic_institution.t) -> match item.logo with | None -> () | Some { path; alt } ->
+      let alt = match alt with | Some x -> x | None -> "" in %>
       <a href="<%s item.url %>" target="_blank" class="p-12 hover:text-primary-600 flex items-center flex-col">
-        <img src="<%s logo %>" class="m-auto" alt="<%s item.name %> logo" />
+        <img src="<%s path %>" class="m-auto" alt="<%s alt %>" />
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
        <img src="<%s path %>" class="m-auto" alt="<%s Option.value ~default:"" alt %>" />
```

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 12]
```diff
@@ -39,34 +39,38 @@ Layout.render
...
         <div class="font-bold mt-8"><%s item.name %></div>
       </a>
       <% ); %>
     </div>
     <div :class="continent != 'Europe' ? 'hidden' : 'grid-logos'">
-      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "Europe") |> List.iter (fun (item : Ood.Academic_institution.t) -> let logo = match item.logo with | Some x -> "/media/" ^ x | None -> "" in %>
+      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "Europe") |> List.iter (fun (item : Ood.Academic_institution.t) -> match item.logo with | None -> () | Some { path; alt } ->
+        let alt = match alt with | Some a -> a | None -> "" in %>
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "Europe")
               |> List.iter (fun (item : Ood.Academic_institution.t) -> match item.logo with | None -> () | Some { path; alt } -> %>
```

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 13]
```diff
@@ -39,34 +39,38 @@ Layout.render
...
-        <img src="<%s logo %>" class="m-auto" alt="<%s item.name %> logo" />
+        <img src="<%s path %>" class="m-auto" alt="<%s alt %>" />
         <div class="font-bold mt-8"><%s item.name %></div>
       </a>
       <% ); %>
     </div>
     <div :class="continent != 'Asia' ? 'hidden' : 'grid-logos'">
-      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "Asia") |> List.iter (fun (item : Ood.Academic_institution.t) -> let logo = match item.logo with | Some x -> "/media/" ^ x | None -> "" in %>
+      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "Asia") |> List.iter (fun (item : Ood.Academic_institution.t) -> match item.logo with | None -> () | Some { path; alt } ->
+      let alt = match alt with | Some x -> x | None -> "" in %>
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "Asia")
               |> List.iter (fun (item : Ood.Academic_institution.t) -> match item.logo with | None -> () | Some { path; alt } -> %>
```

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 14]
```diff
@@ -39,34 +39,37 @@ Layout.render
       </div>
     </div>
     <div :class="continent != 'All' ? 'hidden' : 'grid-logos'">
-      <% users |> List.iter (fun (item : Ood.Academic_institution.t) -> let logo = match item.logo with | Some x -> "/media/" ^ x | None -> "" in %>
+      <% users |> List.iter (fun (item : Ood.Academic_institution.t) -> match item.logo with | None -> () | Some { path; alt } -> %>
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
      <% users |> List.iter (function ({ logo = Some { path; alt}; url; name; _ } : Ood.Academic_institution.t) -> %>
```

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 15]
```diff
@@ -39,34 +39,37 @@ Layout.render
...
       <a href="<%s item.url %>" target="_blank" class="p-12 hover:text-primary-600 flex items-center flex-col">
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
      <a href="<%s url %>" target="_blank" class="p-12 hover:text-primary-600 flex items-center flex-col">
```

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 16]
```diff
@@ -39,34 +39,37 @@ Layout.render
...
-        <img src="<%s logo %>" class="m-auto" alt="<%s item.name %> logo" />
+        <img src="<%s path %>" class="m-auto" alt="<%s Option.value ~default:"" alt %>" />
         <div class="font-bold mt-8"><%s item.name %></div>
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
        <div class="font-bold mt-8"><%s name %></div>
```

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 17]
```diff
@@ -39,34 +39,37 @@ Layout.render
...
       </a>
       <% ); %>
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
      <% | _ -> ()); %>
```

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 18]
```diff
@@ -39,34 +39,37 @@ Layout.render
...
     </div>
 
     <div :class="continent != 'North America' ? 'hidden' : 'grid-logos'">
-      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "North America") |> List.iter (fun (item : Ood.Academic_institution.t) -> let logo = match item.logo with | Some x -> "/media/" ^ x | None -> "" in %>
+      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "North America")
+               |> List.iter (fun (item : Ood.Academic_institution.t) -> match item.logo with | None -> () | Some { path; alt } -> %>
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "North America")
               |> List.iter ((function ({ logo = Some { path; alt}; url; name; continent = "North America"; _ } : Ood.Academic_institution.t) -> %>
```

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 19]
```diff
@@ -39,34 +39,37 @@ Layout.render
...
       <a href="<%s item.url %>" target="_blank" class="p-12 hover:text-primary-600 flex items-center flex-col">
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
      <a href="<%s url %>" target="_blank" class="p-12 hover:text-primary-600 flex items-center flex-col">
```

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 20]
```diff
@@ -39,34 +39,37 @@ Layout.render
...
-        <img src="<%s logo %>" class="m-auto" alt="<%s item.name %> logo" />
+        <img src="<%s path %>" class="m-auto" alt="<%s Option.value ~default:"" alt %>" />
         <div class="font-bold mt-8"><%s item.name %></div>
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
        <div class="font-bold mt-8"><%s name %></div>
```

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 21]
```diff
@@ -39,34 +39,37 @@ Layout.render
...
       </a>
       <% ); %>
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
      <% | _ -> ()); %>
```

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

**Comment by cuihtlauac** [comment]
Inline suggestions should also be applied to 'Europe' and 'Asia'

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 22]
```diff
@@ -39,37 +39,40 @@ Layout.render
...
-      <% ); %>
+      <% | _ -> ()); %>
     </div>
     <div :class="continent != 'Asia' ? 'hidden' : 'grid-logos'">
-      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "Asia") |> List.iter (fun (item : Ood.Academic_institution.t) -> let logo = match item.logo with | Some x -> "/media/" ^ x | None -> "" in %>
-      <a href="<%s item.url %>" target="_blank" class="p-12 hover:text-primary-600 flex items-center flex-col">
-        <img src="<%s logo %>" class="m-auto" alt="<%s item.name %> logo" />
-        <div class="font-bold mt-8"><%s item.name %></div>
+      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "Asia")
+               |> List.iter (function ({ logo = Some { path; alt }; url; name; continent = "Asia"; _ } : Ood.Academic_institution.t) -> %>
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
      <% users |> List.iter (function ({ logo = Some { path; alt }; url; name; continent = "Asia"; _ } : Ood.Academic_institution.t) -> %>
```

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 23]
```diff
@@ -39,37 +39,40 @@ Layout.render
...
-      <% ); %>
+      <% | _ -> ()); %>
     </div>
     <div :class="continent != 'Europe' ? 'hidden' : 'grid-logos'">
-      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "Europe") |> List.iter (fun (item : Ood.Academic_institution.t) -> let logo = match item.logo with | Some x -> "/media/" ^ x | None -> "" in %>
-      <a href="<%s item.url %>" target="_blank" class="p-12 hover:text-primary-600 flex items-center flex-col">
-        <img src="<%s logo %>" class="m-auto" alt="<%s item.name %> logo" />
-        <div class="font-bold mt-8"><%s item.name %></div>
+      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "Europe")
+               |> List.iter (function ({ logo = Some { path; alt }; url; name; continent = "Europe"; _ } : Ood.Academic_institution.t) -> %>
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
      <% users |> List.iter (function ({ logo = Some { path; alt }; url; name; continent = "Europe"; _ } : Ood.Academic_institution.t) -> %>
```

File: src/ocamlorg_frontend/pages/academic_users.eml [hunk 24]
```diff
@@ -39,37 +39,40 @@ Layout.render
...
+      <% | _ -> ()); %>
     </div>
 
     <div :class="continent != 'North America' ? 'hidden' : 'grid-logos'">
-      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "North America") |> List.iter (fun (item : Ood.Academic_institution.t) -> let logo = match item.logo with | Some x -> "/media/" ^ x | None -> "" in %>
-      <a href="<%s item.url %>" target="_blank" class="p-12 hover:text-primary-600 flex items-center flex-col">
-        <img src="<%s logo %>" class="m-auto" alt="<%s item.name %> logo" />
-        <div class="font-bold mt-8"><%s item.name %></div>
+      <% users |> List.filter (fun x -> x.Ood.Academic_institution.continent = "North America")
+               |> List.iter (function ({ logo = Some { path; alt }; url; name; continent = "North America"; _ } : Ood.Academic_institution.t) -> %>
```
Comment (cuihtlauac) [inline src/ocamlorg_frontend/pages/academic_users.eml]:
```suggestion
      <% users |> List.iter (function ({ logo = Some { path; alt }; url; name; continent = "North America"; _ } : Ood.Academic_institution.t) -> %>
```

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

**Comment by cuihtlauac** [comment]
@Zineb-Ada: Can we imagine a way to tweak the yaml parsing code in order to support both the old and new syntaxes? That would allow not avoid touching the yaml files which do not have alt text added.

**Comment by Zineb-Ada** [comment]
@cuihtlauac I will work it out.

**Comment by tmattio** [comment]
We don't seem to be working on this anymore, so closing to reduce noise, but feel free to reopen

**Pull request closed without merging**